java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S2070_java.html

```diff
@@ -39,5 +39,5 @@ <h2>See</h2>
   <li> <a href="http://shattered.io/">SHAttered</a> - The first concrete collision attack against SHA-1. </li>
 </ul>
 <h2>Deprecated</h2>
-<p>This rule is deprecated; use {rule:squid:S4790} instead.</p>
+<p>This rule is deprecated; use {rule:squid:S4790}, {rule:squid:S5344} instead.</p>
```

Comment:
Implemented in SonarJava 5.13 ([SONARJAVA-3104](https://jira.sonarsource.com/browse/SONARJAVA-3104))